### PR TITLE
RD-5448 Replace marketplace default table mode to catalog mode

### PR DESCRIPTION
**Pull request by PatrycjaKubiczek**
## Description

RD-5448 
change catalog marketplace default mode from table to catalog

## Screenshots / Videos

n/a

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3562/)

## Documentation

n/a

### CODE REVIEW

File: templates/pages/blueprintMarketplace.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
                             "definition": "blueprintCatalog",
                             "configuration": {
                                 "jsonPath": "https://repository.cloudifysource.org/cloudify/blueprints/6.4/getting_started.json",
-                                "displayStyle": "table",
+                                "displayStyle": "catalog",
```
Comment (qooban) [inline templates/pages/blueprintMarketplace.json:18]: As `displayStyle` default value is right now `catalog` I think we can remove L18 completely, it should automatically be set by widget framework.

The same goes to other lines with `"displayStyle":..." in this file.

**Review by qooban — COMMENTED** [review]
Change looks good. I added just one comment to even make it simpler and more robust.

File: templates/pages/blueprintMarketplace.json (hunk 1 above)
Comment (PatrycjaKubiczek) [reply templates/pages/blueprintMarketplace.json]: removed lines with displayStyle: https://github.com/cloudify-cosmo/cloudify-stage/pull/2175/commits/718680c6d0fc978c84bb62ddf6bb848f7c7de303

**Review by qooban — APPROVED** [review]
All good 👍🏻

**Pull request merged**